uiowa/uiowa
# [ITS] Alert email settings

Resolves #7617 





# How to test


```
ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli /admin/config/sitenow/its-settings && ddev launch -m
```
Email settings should be empty
Go to an alert and send an alert notification; no email should be sent to mailpit
`ddev drush @its.local its-digest`
You should see a relevant message that no email was sent; no email should be sent to mailpit

Masquerade as a non-webmaster, and see that you can't edit the settings at /admin/config/sitenow/its-settings
Masquerade as a webmaster and visit the /admin/config/sitenow/its-settings
Set email(s) for the various options
Re-test sending notification and digest emails; emails should be sent to mailpit with their respective to/bcc addresses
